CMakeLists.txt

```diff
@@ -65,6 +65,11 @@ if(NOEGL)
     add_definitions(-DNOX11)
 endif()
 
+#DEFAULT_ES=2
+if(${DEFAULT_ES} STREQUAL 2)
```

Comment:
You need to put all inside quotes to allow empty argument:
if("${DEFAULT_ES}" STREQUAL "2")